### ISSUE TITLE
Go to References not working case

### ISSUE THREAD

**Issue by Chupaka** [hashicorp/terraform-ls#585]
### Server Version

```
terraform-ls 0.19.0
```

### Terraform Version

```
Terraform v1.0.2
on darwin_amd64
+ provider registry.terraform.io/hashicorp/google v3.75.0
```

### Client Version

```
VS Code 1.58.2
HashiCorp Terraform v2.13.1
```

### Terraform Configuration Files

```hcl
variable "test2" {
  default = "mynameis"
}

resource "google_compute_instance" "checking" {
  name = var.test2
  machine_type = "fake"
  network_interface {}
  boot_disk {
    initialize_params {
      image = "img"
    }
  }
}
```

### Log Output
```
2021/07/16 14:36:50 server.go:591: Received 1 new requests
2021/07/16 14:36:50 server.go:175: Processing 1 requests
2021/07/16 14:36:50 server.go:266: Checking request for "textDocument/references": {"textDocument":{"uri":"file:///Users/pavel.skuratovich/tmp/go-to-reference/file1.tf"},"position":{"line":1,"character":22},"context":{"includeDeclaration":true}}
2021/07/16 14:36:50 rpc_logger.go:29: Incoming request for "textDocument/references" (ID 452): {"textDocument":{"uri":"file:///Users/pavel.skuratovich/tmp/go-to-reference/file1.tf"},"position":{"line":1,"character":22},"context":{"includeDeclaration":true}}
2021/07/16 14:36:50 provider_schema.go:196: PSS: getting provider schema (/Users/pavel.skuratovich/tmp/go-to-reference, registry.terraform.io/-/google, )
2021/07/16 14:36:50 references.go:60: finding origins for inner-most target: &lang.ReferenceTarget{Addr:lang.Address{lang.RootStep{Name:"var"}, lang.AttrStep{Name:"test2"}}, ScopeId:"variable", RangePtr:(*hcl.Range)(0xc009870500), Type:cty.String, Name:"", Description:lang.MarkupContent{Value:"", Kind:0x0}, NestedTargets:lang.ReferenceTargets(nil)}
2021/07/16 14:36:50 rpc_logger.go:50: Response to "textDocument/references" (ID 452): []
2021/07/16 14:36:50 server.go:247: Completed 1 requests [3.362738ms elapsed]
```

### Steps to Reproduce

Click line 1-3 and select "Go to References"

### Expected Behavior

Jump to line 6 (`name = var.test2`)

### Actual Behavior

`No references found for 'test2'`

**Comment by radeksimko** [comment]
Reproduced, thank you for the detailed report.

I will look for the root cause and fix.

**Comment by radeksimko** [comment]
This was fixed as part of #587 which is part of the release `0.19.1` I just published.

Can you try updating to that version and confirm the problem is resolved? VS Code should prompt the upgrade automatically upon launch or within the next 24 hours.

**Comment by Chupaka** [comment]
Thanks, I confirm that this case was fixed, but I'm filing another one right now :)
